translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="119"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehl + Enter (z.B. help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <source>Daemon doesn&apos;t appear to be running</source>
-        <translation type="vanished">Daemon scheint nicht aktiv zu sein</translation>
+        <translation type="vanished">Dämon scheint nicht aktiv zu sein</translation>
```

Comment:
Wikipedia starts its respective entry with "Als Daemon oder Dämon (auch häufig in der Schreibweise Demon)". So any way of spelling is okay, we just have to think about which is more beautiful. In my opinion, when there is a German word, we shouldn't be shy using it. Most people not familiar with IT processes would guess that the "ae" means "ä" anyway - my mother might even believe it's a bad translation and wonder why she should let an evil spirit in her computer! ;) But in the end, I don't care. I just want to make progress in the project.